src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +27,51 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: mapboxLayoutAtributes.maxzoom,
+        radius: {
+            role: 'info',
+            valType: 'number',
+            dflt: 50,
+            description: 'Radius of each cluster when clustering points.',
+        },
+        step: {
+            role: 'info',
+            valType: 'number',
+            arrayOk: true,
+            dflt: -1,
+            min: -1,
+            description: [
+                'Sets the steps for each cluster.'
+            ].join(' ')
+        },
+        size: {
+            role: 'info',
+            valType: 'number',
+            arrayOk: true,
+            dflt: 20,
+            min: 0,
+            description: [
+                'Sets the size for each cluster step.'
+            ].join(' ')
+        },
+        color: {
+            valType: 'color',
+            arrayOk: true,
+            role: 'style',
+            description: [
+                'Sets the color for each cluster step.'
+            ].join(' ')
+        },
+        opacity: markerAttrs.opacity,
```

Comment:
We should provide a `dflt` for this attribute:
```js
opacity: extendFlat({}, markerAttrs.opacity, {
   dflt: 1
})
```
otherwise the cluster points would be invisible.